File: csharp/bypstest-ser/src-ser/com/wilutions/byps/test/api/inherit/PlantService.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using com.wilutions.byps;

namespace com.wilutions.byps.test.api.inherit
{
	
	// checkpoint com.wilutions.byps.gen.cs.GenRemoteClass:93
	public interface PlantService : BRemote {
	
		String Grow();
		void Grow(BAsyncResult<String> asyncResult) ;
		Task<String> GrowAsync();
		
		
	}
}  // end namespace

File: java/bypstest-api/src/byps/test/api/inherit/PlantService.java
package byps.test.api.inherit;

import byps.BRemote;
import byps.RemoteException;

/**
 * 
 * @BSessionParamType com.wilutions.byps.test.api.auth.SessionInfo
 *
 */
public interface PlantService extends BRemote {
  public String grow() throws RemoteException;
}
